workspace/utilities/lib/svg-icons.xsl

```diff
@@ -1,45 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why delete this ?

Reply:
Has not been used since 1983. But mostly because we also had site-svg and one of them was unnecessary. Chose this one.